src/opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -88,6 +88,7 @@ namespace {
         0.0,
         0.0,
         0.0,
+        0.0,
```

Comment:
New unit - typically something which should go in a separate PR as soon as the need is identified.

Reply:
I see the point, however, on the other hand this change is strongly connected the need to use this new unit conversion in this PR and the combination also gives a test that the code for the conversion actually works OK. Hence I hope they can be combined in this case.

Reply:
My point is just that these changes could have been sent out at an earlier stage in the development process. That would have made the main PR somewhat smaller - and that is always good. No reason to split it off now.